src/components/Layout/ActivityBar/Profile.tsx

```diff
@@ -0,0 +1,42 @@
+import { PersonIcon } from '@radix-ui/react-icons'
+import { Dialog, Flex, IconButton, Tooltip } from '@radix-ui/themes'
+import { Profile as ProfileContent } from '@/components/Profile'
+import { useState } from 'react'
+
+function ProfileDialog({
+  open,
+  onOpenChange,
+}: {
+  open: boolean
+  onOpenChange: (open: boolean) => void
+}) {
+  return (
+    <Dialog.Root open={open} onOpenChange={onOpenChange}>
```

Comment:
<img width="653" alt="image" src="https://github.com/user-attachments/assets/6ba3b107-8d60-4b11-9049-f62720ee8415" />

I was a bit puzzled at first, because I expected to see a close button somewhere. One can argue that our users should know that clicking outside/hitting `esc` closes a dialog, but I think it should be more explicit (I'm actually surprised radix dialogs don't have the `x` button :/)